drivers/hmis/app/graphql/types/hmis_schema/external_form_submission.rb

```diff
@@ -20,10 +20,14 @@ class HmisSchema::ExternalFormSubmission < Types::BaseObject
     field :status, HmisSchema::Enums::ExternalFormSubmissionStatus, null: false
     field :notes, String, null: true
     field :definition, Forms::FormDefinition, null: false
-    custom_data_elements_field
+    field :values, GraphQL::Types::JSON, null: true
 
     def definition
       load_ar_association(object, :definition)
     end
+
+    def values
+      object.cleaned_values
```

Comment:
This will be nil if the submission was created before the migration introduced in this PR. I debated returning `|| object.raw_data` here, but it would only case an error later on when the form processor encounters an unexpected link ID

Reply:
For backwards compat, we could also consider displaying the CDEs here if they already exist, but this seems like more complexity than it's worth to me.

Reply:
This is easier now that we are getting rid of `cleaned_values`, no need to migrate

Reply:
For displaying the form in dynamic view, always use the raw values, regardless of whether it's been processed. Add comment sharing example of changing client name in HMIS